cache/edge_lambdas/src/toggler.ts

```diff
@@ -30,6 +30,14 @@ export let tests: Test[] = [
       return !!request.uri.match(/\/works\/.*$/);
     },
   },
+  {
+    id: 'newTags',
+    title: 'A/B test for new tags',
+    range: [0, 100],
```

Comment:
How does this work? (sorry if it's documented!)

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/blob/main/cache/edge_lambdas/src/toggler.ts#L11-L13

I think it's maybe _slightly_ over-engineered to allow us to put people in different conditions if/when we're running more than one A/B test at once (which we never have and probably won't be in a hurry)

Reply:
Thanks for that! yeah it looks like it complicates things for sure.